saskatoon/unittests/init

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Create the testing fixture .json file from initial data in the 'fixtures' folder
+
+# automatically exit on error
+set -e
+
+# get script directory no matter where it's called from (doesn't work w/ symlinks)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd $SCRIPT_DIR
+
+# rename the current db and restore it at the end of process
+if [ -e ../db.sqlite3 ]; then
+    mv ../db.sqlite3 ../db_temp.sqlite3
+fi
```

Comment:
I don't think you can assume that it's a sqlite3 file and it's not necessarily located in `saskatoon` folder
https://github.com/LesFruitsDefendus/saskatoon-ng/blob/main/saskatoon/env.template#L20